### PR TITLE
Update SwiftFlutterJailbreakDetectionPlugin.swift

### PR DESCRIPTION
**Pull request by oscarshaitan**
Check if we are on a M1 device, we are return false to jailbroken

code base on 
[https://developer.apple.com/forums/thread/656122?answerId=651150022#651150022](https://developer.apple.com/forums/thread/656122?answerId=651150022#651150022)

### CODE REVIEW

File: ios/Classes/SwiftFlutterJailbreakDetectionPlugin.swift [hunk 1]
```diff
@@ -8,20 +8,22 @@ public class SwiftFlutterJailbreakDetectionPlugin: NSObject, FlutterPlugin {
...
-        default:
-            result(FlutterMethodNotImplemented)
+    switch call.method {
+      case "jailbroken":
+        if #available(iOS 14.0, *) {
+          result(!ProcessInfo.processInfo.isiOSAppOnMac)
+        }
+        else{
+          let isJailBroken = DTTJailbreakDetection.isJailbroken()
+          result(true)
```
Comment (jeroentrappers) [inline ios/Classes/SwiftFlutterJailbreakDetectionPlugin.swift:19]: result(true) is not correct.

**Comment by jeroentrappers** [comment]
Please try to get this change merged upstream. I will update the reference to DTTJailbreakDetection once it has been implemented there.

There might be a good reason for people who want jailbreak protection to not allow their apps to run on M1 macs. These devices allow debugging and analysis of the apps, This is exactly what is to be avoided. So I'm quite reluctant to allow this.

File: ios/Classes/SwiftFlutterJailbreakDetectionPlugin.swift (hunk 1 above)
Comment (oscarshaitan) [reply ios/Classes/SwiftFlutterJailbreakDetectionPlugin.swift:19]: o.O you are right!!!

**Comment by jeroentrappers** [comment]
Won't fix

**Pull request closed without merging**